### PR TITLE
Always add account and region to the response

### PR DESCRIPTION
**Pull request by sergeymirzoyan1991**

### CODE REVIEW

File: src/c7n-executor.ts [hunk 1]
```diff
@@ -105,31 +99,29 @@ export class C7nExecutor {
...
-            result = result.map(data => {
-              data.C8rAccount = currentAccount + ' - Current'
+            result = C7nExecutor.fetchResourceJson(C7nExecutor.buildResourcePath(dir, policyName)).map(data => {
+              data.C8rAccount = currentAccount
               return data
             })
           }
+        }
+
+        if (useMultiAccount) { /// manaca account
```
Comment (marashlyansuren) [inline src/c7n-executor.ts:114]: let's remove comment, or write English please

File: src/domain/types/aws/eip.ts [hunk 2]
```diff
@@ -8,4 +8,8 @@ export class Eip extends ProviderResource {
     readonly _c8rRegion?: string,
     readonly _c8rAccount?: string
   ) { super() }
+
+  getRegion (): string {
```
Comment (marashlyansuren) [inline src/domain/types/aws/eip.ts:12]: @sergeymirzoyan1991 let's have abstract function in ProviderResource

**Review by marashlyansuren — COMMENTED** [review]
Small comments

File: src/domain/types/aws/eip.ts (hunk 2 above)
Comment (sergeymirzoyan1991) [reply src/domain/types/aws/eip.ts:12]: added

File: src/c7n-executor.ts (hunk 1 above)
Comment (sergeymirzoyan1991) [reply src/c7n-executor.ts]: removed

**Review by marashlyansuren — APPROVED** [review]

**Pull request merged**